robot/physics.py

```diff
@@ -0,0 +1,63 @@
+import math
+from pyfrc.physics import motor_cfgs, tankmodel
+from pyfrc.physics.units import units
+
+
+class PhysicsEngine(object):
+    """
+    Simulates a 4-wheel robot using Tank Drive joystick control
+    """
+
+    def __init__(self, physics_controller):
+        """
+        :param physics_controller: `pyfrc.physics.core.Physics` object
+                                       to communicate simulation effects to
+        """
+
+        self.physics_controller = physics_controller
+
+        # Change these parameters to fit your robot!
+
+        self.drivetrain = tankmodel.TankModel.theory(
+            motor_cfgs.MOTOR_CFG_CIM,           # motor configuration
+            110 * units.lbs,                      # robot mass
```

Comment:
I know I'm really nitpicking hard here (and this is probably from the example you used) but pounds are a unit of weight, not mass.